### ISSUE TITLE
NTR: Deoxyuridine high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8811]
New term request Deoxyuridine high in urine (segal_050428154938):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000116
- label: Elevated urinary deoxyuridine level
- parents: [Abnormal urinary pyrimidine level (HP:0033162)](https://hpo.jax.org/app/browse/term/HP:0033162)
- pmid: 
- def: The amount of deoxyuridine in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8811
- synonyms: Deoxyuridine high in urine(NONE)
